CODEOWNERS

```diff
@@ -2,7 +2,47 @@
 /erroranalysis @gaugup @imatiach-msft
 
 # Responsible AI widgets package
-/raiwidgets @imatiach-msft @xuke444 @zhb000 @vinuthakaranth @RubyZ10 @tongyu-microsoft @romanlutz
+/raiwidgets @imatiach-msft @xuke444 @romanlutz
 
 # Responsible AI SDK package
-/responsibleai @gaugup @imatiach-msft @riedgar-ms @xuke444 @kicha0
\ No newline at end of file
+/responsibleai @gaugup @imatiach-msft @riedgar-ms @xuke444 @kicha0
+
+# Responsible AI SDK package
+/responsibleai @gaugup @imatiach-msft @riedgar-ms @xuke444 @kicha0
```

Comment:
There seems to be two lines for responsibleai here. Is that needed?

Reply:
my bad

Reply:
I still the duplicate line for responsibleai. Can you recheck?

Reply:
just pushed